toolchain/mfc/args.py

```diff
@@ -181,3 +182,4 @@ def add_common_arguments(p, mask = None):
             args[e] = os.path.abspath(args[e])
 
     return args
+git clone https://github.com/aricer123/MFC.git
```

Comment:
?

Reply:
https://github.com/MFlowCode/MFC/pull/605#issuecomment-2326927176
Resolved in the most recent commit